build.sh

```diff
@@ -24,7 +24,7 @@ $EM_DIR/emconfigure ./configure \
     --disable-workshop \
     --disable-netbeans \
     --disable-sniff \
-    --disable-multibyte \
+    --enable-multibyte \
```

Comment:
Sorry, this was not supposed to be committed. The size increases from 25MB to 44MB with multibyte, but the multibyte patch is not complete since it does not work yet. I shall send a different pull request for that once I am done.